### DISCUSSION TITLE
Network Issue with Routers

### DISCUSSION THREAD

**Discussion by seaverd** [Koenkk/zigbee2mqtt#5585] [General]
I could use some assistance.  I am having an issue with my zigbee2mqtt network as it relates to router type devices.  I am running zigbee2mqtt version 1.7 in docker.  My coordinate is the zzh stick with the 10/26/2020 firmware.  All of my network endpoint devices are Xiaomi (motion, button, scene controller, temp, water leak detector, etc)  

When my network is just the coordinator paired with the Xiaomi devices, everything works fine.  However, I have several devices that have low signal quality, so I decided to add some devices with routing capabilities.  These devices are my new Sinope electric radiant floor thermostat, an Ikea RGB bulb and two Ikea outlets.    These routing devices add to my network fine, but when I then add a new endpoint the endpoint connects to one of the routers and works for say an hour of so and then drops off the network.  See attached network map...all of the unconnected devices were connected to one of the routers and then stopped working.  
![zigbee2mqttnetworkmap](https://user-images.githubusercontent.com/2743685/103690884-5c126c80-4f63-11eb-97ab-2ca3fefcf0df.jpg)I am pulling my hair out, have blown out my zigbee2mqtt docker, deleted retained mqtt messages, reset routing devices, created a new network under a new pan_id and just want to get everything working.  

 Is there a best practice for starting from scratch and rebuilding the network...what order should they be paired.  If need be, I would be fine getting rid of the Ikea devices if they are the issue but the floor thermostat is too expensive and necessary  Any suggestions would be greatly appreciated!

**Comment by Pare1984** [comment]
I have a similar problem. All devices are connected to one router although another router is much closer and therefore probably has a better connection quality. How can I re-route the controller?
